packages/fxa-settings/src/components/Settings/PageDeleteAccount/index.tsx

```diff
@@ -146,7 +147,7 @@ export const PageDeleteAccount = (_: RouteComponentProps) => {
           'flow.settings.account-delete',
           'confirm-password.success'
         );
-        hardNavigateToContentServer(`${ROOTPATH}?delete_account_success=true`);
```

Comment:
The new api is harder to read in one way: it's not obvious what that last `true` argument is for.